### ISSUE TITLE
solve : [collapsiblesidebar] Doesn’t work with “stickysidebar” in classic theme.

### ISSUE THREAD

**Issue by sakazu310** [sphinx-doc/sphinx#4500]
In the [document](http://www.sphinx-doc.org/en/stable/theming.html),
```
collapsiblesidebar : "Doesn’t work with stickysidebar".
```

However, we can use both by adding following: 

In **conf.py** [html_theme_options] ,
``` 
"collapsiblesidebar" : "true",
```

In **classic.css**
``` css:classic.css
margin-left : 0,
position:fixed
```

**Comment by tk0miya** [comment]
Could you send us a PR please?
Then I'll check it later.

**Comment by tk0miya** [comment]
I confirmed it works well without any modification.

**Comment by tk0miya** [comment]
Fixed by #5202.
Thanks,
